scripts/ci/test_plan.py

```diff
@@ -159,12 +162,14 @@ def find_boards(self):
             if p and p.groups():
                 boards.add(p.group(1))
 
+        # Limit search to $ZEPHYR_BASE since this is where the changed files are
+        lb_args = argparse.Namespace(**{ 'arch_roots': [], 'board_roots': [] })
```

Comment:
list_boards should be enhanced to be used from other script without going via argparse, the above seems to be very awkward